frontend/pages/initiative/[id]/relationships.vue

```diff
@@ -28,15 +29,19 @@ const canManage = computed(() => {
   return relationships.value.some((r) => r.userId === mm.id && r.manager)
 })
 
-const makeManager = (userId: string) => {
-  alert('make manager not yet implemented')
-}
-const removeFromInitiative = (userId: string) => {
-  alert('remove from initiative not yet implemented')
-}
-const removeManager = (userId: string) => {
-  alert('remove manager not yet implemented')
-}
+const changeMembership = (userId: string, member: boolean | undefined, manager: boolean | undefined) =>
```

Comment:
Doesn't look like `manager` is used here?

Reply:
Line 44! It's just never used to add someone, but it can be used to remove them :)

Reply:
Lol no no no, you never use `manager` _in this function_, so line 44 ain't doing nothing